inference-engine/tests/functional/plugin/myriad/common/myriad_layers_interpolate_test.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "myriad_layers_interpolate_test.hpp"
+
+// INSTANTIATE_TEST_CASE_P(
```

Comment:
why is it commented?

Reply:
It is not ready yet, I am currently working on tests.